<launch>

	<node
		pkg="lrm_drivers"
		name="teleop"
		type="teleoperation_by_joy"
		output="screen"
	>
		<param
			name="reset_button"
			value="3" />
		<param
			name="brake_axis"
			value="1" />
		<param
			name="brake_scale"
			value="100" />
		<param
			name="steering_axis"
			value="0" />
		<param
			name="steering_scale"
			value="30" /> <!-- max angle in degrees -->
		<param
			name="velocity_increase_button"
			value="6" />
		<param
			name="velocity_decrease_button"
			value="7" />
		<param
			name="velocity_scale"
			value="20" />
	</node>

	<node
		pkg="joy"
		name="joystick"
		type="joy_node"
		output="screen"
	>
		<param
			name="dev"
			value="/dev/input/js0" />
	</node>

<!-- 
	<node
		name="conv_msg_node"
		pkg="lrm_drivers"
		type="nav_message_node"
		output="screen"
	>
	</node>
 -->
 	
</launch>